includes/admin/views/access-plans/access-plan.php

```diff
@@ -15,8 +15,8 @@
 
 defined( 'ABSPATH' ) || exit;
 
-// create a "step" attribute for price fields according to LLMS settings
-$price_step = number_format( 0.01, get_lifterlms_decimals(), get_lifterlms_decimal_separator(), get_lifterlms_thousand_separator() );
+// create a "step" attribute for price fields
+$price_step = '0.01';
```

Comment:
```suggestion
$price_step = number_format( 0.01, get_lifterlms_decimals() );
```
So that we still take into account the user's settings about the number of decimals.